libs/financial-aid/shared/src/lib/interfaces.ts

```diff
@@ -214,6 +214,7 @@ export interface CreateApplication {
   postalCode?: string
   city?: string
   municipalityCode?: string
+  applicationId?: string
```

Comment:
I search this file and sound we are using applicationId in quite a few places, so I think we need a more specific name applicationSystemId to not get confused